.github/workflows/ci.yml

```diff
@@ -51,28 +51,48 @@ jobs:
         env:
           ESP_IDF_VERSION: ${{ matrix.idf-version }}
           ESP_IDF_SDKCONFIG_DEFAULTS: "${{ github.workspace }}/.github/configs/sdkconfig.defaults"
+<<<<<<< HEAD
           RUSTFLAGS: "${{ matrix.idf-version == 'v5.1.2' && '--cfg espidf_time64' || ''}}"
+=======
+          ESP_IDF_COMPONENT_MANAGER: "${{ matrix.idf-version == 'v5.1.1' && 'y' || 'n'}}"
+          RUSTFLAGS: "${{ matrix.idf-version == 'v5.1.1' && '--cfg espidf_time64' || ''}}"
```

Comment:
version 5.1.1 still in use here

Reply:
The code in the CI yaml will all be reverted. This example is prepared to only compile for ESP IDF 4. There are comments in the example if the user wants to build it for ESP IDF 5. But we cannot alter the `Cargo.toml` file of `esp-idf-svc` in a way where it becomes compatible with ESP IDF 5 only (by including the websocket component). The user will have to do this in their own `Cargo.toml.